docs-src/concepts/what-is-an-activity-execution.md

```diff
@@ -23,6 +23,15 @@ You can customize [Activity Execution timeouts](/concepts/what-is-a-start-to-clo
 
 If an Activity Execution fails (because it exhausted all retries, threw a [non-retryable error](/concepts/what-is-a-retry-policy#non-retryable-errors), or was canceled), the error is returned to the [Workflow](/workflows), which decides how to handle it.
 
+:::note
```

Comment:
The above "By default, an Activity Execution has no time limit." in incorrect. The timeout is required.